src/test/java/hw6/BaseClassTest.java

```diff
@@ -0,0 +1,45 @@
+package hw6;
+
+import hw6.service.webdriver.WebDriverSingleton;
+import hw6.steps.ActionStep;
+import hw6.steps.AssertionStep;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.concurrent.TimeUnit;
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.ITestContext;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+public class BaseClassTest {
+    protected WebDriver webDriver;
+    protected Configuration properties;
+    private final String pathToProperties = "src/test/resources/hw3Properties/pageData.properties";
```

Comment:
Why properties are in `hw3` folder?

Reply:
This data was used in the same way in all works (starting with 3), so it was decided not to duplicate this file.